Code.Rmd

```diff
@@ -7,19 +7,30 @@ source("common.R")
 # R code {#r}
 
 The first principle of using a package is that all R code goes in `R/`.
-In this chapter, you'll learn about the `R/` directory, my recommendations for organising your functions into files, and some general tips on good style.
-You'll also learn about some important differences between functions in scripts and functions in packages.
+In this chapter, you'll learn about the `R/` directory, our recommendations for organising your functions into files, and some general tips on good style.
+You'll also learn some important differences between defining a function in a script and defining it in a package.
 
 ## R code workflow {#r-workflow}
 
-The first practical advantage to using a package is that it's easy to re-load your code.
-You can either run `devtools::load_all()`, or in RStudio press __Ctrl/Cmd + Shift + L__,  which also saves all open files, saving you a keystroke.
+As your use of R becomes more sophisticated, it's common to start to write your own R functions.
+If a function is only used in one place, you probably define it right there.
+But if you've bothered to write a function, it's likely you want to reuse it in multiple places: within one script, across multiple scripts, or even across multiple projects.
+
+The next logical step is to collect such function definitions in one or more dedicated `.R` files and then `source()` them as needed.
+Typically these functions co-evolve with the code where you use them, i.e. your analysis code, your Shiny apps, or your R Markdown reports.
+
```

Comment:
I felt like this needed more of a intro to set the stage.

Something that felt very *implicit* before is that the user has "their code" and that cleaves into 2 types of code: top-level code that carries out their analysis (or whatever) and functions they have defined. And it's the *latter* that we're proposing to store in a package.